diagnostic.md

```diff
@@ -5,74 +5,86 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The back end is the the database or model in MVC terminology. Without it, your
+would not be able to save or commit any data to the back end data store or base. Think of Amazon.com that is not attached to a datbase. Not possible to function.
```

Comment:
Good the backend allows us to store data and have users interact with each other. Careful comparing the model to the database though, they are not the same thing.